### PR TITLE
TE-927:  expose string props for Rates

### PR DESCRIPTION
**Pull request by stephenkr**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-927)

### What **one** thing does this PR do?
Expose string props for `Rates`

__Preview__
screen shot 2018-08-28 at 14 12 28

### CODE REVIEW

File: src/components/property-page-widgets/Rates/component.js [hunk 1]
```diff
@@ -115,6 +133,8 @@ Component.propTypes = {
   ).isRequired,
   /** The headings for each column of the table */
   rateHeadings: PropTypes.arrayOf(PropTypes.string).isRequired,
+  /** The heading text for the room type input */
+  roomTypeHeadingText: PropTypes.string,
```
Comment (marcdavi-es) [inline src/components/property-page-widgets/Rates/component.js]: This is more of a label than a heading, no?

I suggest `roomTypeInputLabel`

File: src/components/property-page-widgets/Rates/component.js [hunk 2]
```diff
@@ -98,6 +114,8 @@ Component.propTypes = {
   onChangeCurrency: PropTypes.func,
   /** A function called when the room type value changes. */
   onChangeRoomType: PropTypes.func,
+  /** The text to display for the price per extra guest */
+  pricePerExtraText: PropTypes.string,
```
Comment (marcdavi-es) [inline src/components/property-page-widgets/Rates/component.js]: This creates inconsistency.

We have `rateCategories.costPerExtraGuest` already.

This should be `costPerExtraGuestLabel`

File: src/utils/default-strings/constants.js [hunk 3]
```diff
@@ -48,4 +49,5 @@ export const TIME = 'Time';
 export const TIME_ZONE = 'Time Zone';
 export const USE_COUPON_CODE = 'Use the coupon code';
 export const VIEW_MORE = 'View more';
+export const VIEW_RATE_INFORMATION_FOR = 'View Rate Information for:';
```
Comment (Jonnygwi) [inline src/utils/default-strings/constants.js]: This should not have the colon. That needs to be extracted out into a helper. `getStringWithColonSuffix` already exists.

File: src/components/property-page-widgets/Rates/component.js (hunk 2 above)
Comment (stephenkr) [reply src/components/property-page-widgets/Rates/component.js]: I'll fix this.

File: src/components/property-page-widgets/Rates/utils/buildPricePerExtraGuestString.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 /**
  * @param {String} cost
+ * @param {String} pricePerExtraText
  * @return {String}
  */
-export const buildPricePerExtraGuestString = cost =>
-  `${cost} / Price Per extra per.`;
+export const buildPricePerExtraGuestString = (cost, pricePerExtraText) =>
+  `${cost} / ${pricePerExtraText}`;
```
Comment (Jonnygwi) [inline src/components/property-page-widgets/Rates/utils/buildPricePerExtraGuestString.js]: Do you mean to remove the full stop?

File: src/utils/default-strings/constants.js (hunk 3 above)
Comment (stephenkr) [reply src/utils/default-strings/constants.js]: Yep true, i'll rely on a util to append the colon

File: src/components/property-page-widgets/Rates/utils/buildPricePerExtraGuestString.js (hunk 4 above)
Comment (stephenkr) [reply src/components/property-page-widgets/Rates/utils/buildPricePerExtraGuestString.js]: I'll add it back.

**Pull request merged**